test/team_stats_test.rb

```diff
@@ -0,0 +1,178 @@
+require './test/test_helper'
+require './lib/stat_tracker'
+require './lib/stat_parser'
+require 'pry'
+
+class TeamStatsTest < Minitest::Test
+
+    def setup
+      v_small_data_paths = {games: './data/game_very_small.csv',
+                            teams: './data/team_info.csv',
+                            game_teams: './data/game_teams_stats_very_small.csv'}
+
+      small_data_paths = {games: './data/game_small.csv',
+                          teams: './data/team_info.csv',
+                          game_teams: './data/game_teams_stats_small.csv'}
+
+      data_paths = {games: './data/game.csv',
+                    teams: './data/team_info.csv',
+                    game_teams: './data/game_teams_stats.csv'}
+
+      @v_small_data = StatTracker.from_csv(v_small_data_paths)
+      @small_data = StatTracker.from_csv(small_data_paths)
+      @data = StatTracker.from_csv(data_paths)
+    end
+
+  def test_team_info_with_respective_team_attributes
+    expected = {
+                team_id: 3,
+                franchise_id: 10,
+                short_name: "NY Rangers",
+                team_name: "Rangers",
+                abbreviation: "NYR",
+                link: "/api/v1/teams/3"
+                }
+    assert_equal expected, @v_small_data.team_info
+
+    expected = {
+                team_id: 16,
+                franchise_id: 11,
+                short_name: "Chicago",
+                team_name: "Blackhawks",
+                abbreviation: "CHI",
+                link: "/api/v1/teams/16"
+                }
+    assert_equal expected, @small_data.team_info
+
+    expected = {
+                team_id: 18,
+                franchise_id: 34,
+                short_name: "Nashville",
+                team_name: "Predators",
+                abbreviation: "NSH",
+                link: "/api/v1/teams/18"
+                }
+    assert_equal expected, @data.team_info
+  end
+
+  def test_best_season
+    assert_equal 20122013, @v_small_data.best_season(6)
+    assert_equal 20122013, @small_data.best_season(5)
+    assert_equal 20142015, @data.best_season(8)
+  end
+
+  def test_worst_season
+    assert_equal 20122013, @v_small_data.worst_season(3)
+    assert_equal 20122013, @small_data.worst_season(16)
+    assert_equal 20142015, @data.worst_season(5)
+  end
+
+  def test_average_win_percentage
+    assert_equal 1.0, @v_small_data.average_win_percentage(6)
```

Comment:
I know that I organized the data- aggregations by team_id -- However, I wonder if organizing them by teamName might be more readable